tooling/profiler/src/flamegraph.rs

```diff
@@ -17,7 +18,7 @@ use super::opcode_formatter::format_opcode;
 
 #[derive(Debug)]
 pub(crate) struct Sample<F: AcirField> {
-    pub(crate) opcode: AcirOrBrilligOpcode<F>,
+    pub(crate) opcode: Option<AcirOrBrilligOpcode<F>>,
```

Comment:
Is the idea that `None` is also captured in the flamegraph?

Reply:
Yes this is handled in the flamegraph. The folding/sorting process checks whether there is some opcode in the sample. If there is not, it will not be added to the list of location names.